app/views/content/life-as-a-teacher/teaching-as-a-career/from-teaching-intern-to-leadership-roles/_article.html.erb

```diff
@@ -0,0 +1,40 @@
+      <h2>Starting my career progression</h2>
+      <p>I applied for the Head of Physics role during my first year of teaching and was successful. This was a big jump for me and essentially allowed me to have a bigger impact beyond the walls of my classroom, improving provision for all students studying key stage 5 physics and the staff that taught them.</p>
+      <h2>Increased responsibility</h2>
+      <p>In my third year I was promoted to second in department and took on more responsibility. I lead meetings, manage the budget, conduct appraisals, and organise ‘learning walks’ to ensure teaching is consistent across the department. This extra work is reflected in my teaching schedule, with reduced classroom hours so that I can manage all the additional tasks.</p>
+      <p>As I became more experienced and developed my leadership skills, I realised I wanted to continue to progress in leadership so that I could affect decisions at a higher level and have a wider impact on students.</p>
+  </section>
+
+  <section class="col col-720">
+    <%= render Content::QuoteComponent.new(
+    text: "I enjoyed the challenge of not just coming up with something that would work in my classroom, but that would work for other teachers and their students.",
+    large: true
+  ) %>
+  </section>
+
+  <section class="col col-720 col-space-l-top">
+      <h2>Career support</h2>
+      <p>I had a great mentor in senior leadership who talked through how they made their decisions and gave me an idea of how to fulfil that leadership role. It’s been great to be able to go to someone who can give me their perspective. I’ve also found the more informal mentoring opportunities, like having a coffee and a chat with a colleague, useful for exploring my leadership ambitions.</p>
+      <h2>Managing the challenges and making a difference</h2>
+      <p>There are of course challenges to the job. I was 21 when I began teaching and 22 when I became head of department. Managing colleagues that were older and more experienced than me was difficult to begin with, but I can say now, after having stayed at the same school for 6 years and developed my reputation, that it’s not so much of a challenge anymore. I found the best approach was to acknowledge other people’s experience and seek their advice, while still driving forward with a clear vision. When I moved over to the maths department, I had to develop a new management style, but fortunately I already knew the staff well.</p>
```

Comment:
```suggestion
      <p>There are challenges to the job, of course. I was 21 when I began teaching and 22 when I became head of department. Managing colleagues that were older and more experienced than me was difficult to begin with, but I can say now, after having stayed at the same school for 6 years and developed my reputation, that it’s not so much of a challenge any more. I found the best approach was to acknowledge other people’s experience and seek their advice, while still driving forward with a clear vision. When I moved over to the maths department, I had to develop a new management style, but fortunately I already knew the staff well.</p>
```
happy for you to disagree with the suggestions

also - should we break this up? feels like a long para